app/src/main/res/values/strings.xml

```diff
@@ -621,7 +621,7 @@ Signature: %5$s</string>
     <string name="external_calendar">External Calendar</string>
     <string name="add_to_calendar_info">To which calendar do you want to add the event?</string>
     <string name="event_export" translatable="false">Event Export</string>
-    <string name="scanner">Scanner</string>
+    <string name="update_failure_message">Update failed.</string>
```

Comment:
This error message is not very descriptive.

Reply:
I'll remove it, since it is not really used anymore.